### ISSUE TITLE
Increase variety in V1 problems

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/checkit-tbil-la#29]
@StevenGubkin pointed me to the following article suggesting ways to generate new examples of vector spaces:

https://www.maa.org/sites/default/files/0746834238317.di020783.02p0422l.pdf

Basically, as long as you have a bijection between a vector space and a set, this lays out how to translate the operation over to the set to give it the same vector space structure.  So I think we can easily get more variety in our V1 problems by simply looking at various bijections of R^2; my favorite examples being invertible polynomial maps.  I just crunched through one example on paper (for the bijection (x+y^2,y) ), and it works out nice.

**Comment by StevenClontz** [comment]
Of course our V1 doesn't use vector spaces. Would this allow us to create non-vector spaces in a similar way? Basically, we would benefit by having one canonical counterexample for each vector space property, then have some scheme to randomize how that counterexample is presented. We also need to make sure that this randomization simplifies into something that doesn't become computationally tricky.

**Comment by siwelwerd** [comment]
Yep, a couple details to work out but wanted to leave a note here so I didn't forget.  

As an aside, is it possible to have Sage symbolicly verify the properties, rather than listing them ourselves?  This would be aesthetically nicer, but maybe more importantly would eliminate a source of  potential errors.
